src/components/landing/LandingOne.tsx

```diff
@@ -1,3 +1,54 @@
+import styled from "@emotion/styled";
+import Image from "next/image";
+import Link from "next/link";
+
+import { ImgLanding01 } from "../../../public/assets/images";
+import { DefaultButton } from "../common/styled/Button";
+
 export default function LandingOne() {
-  return <div>LandingOne</div>;
+  return (
+    <>
+      <StWrapper>
+        <StArticle>
+          <StTitle>
+            진짜 독서가들을 위한
+            <br />
+            독서법을 만들어 갑니다
+          </StTitle>
+          <Link href="/main">
```

Comment:
```suggestion
          <Link href="/main" passHref>
```

맞는지는 모르겠지만 `passHref` 넣어주기?!
[참고 블로그](https://f-dever-error-log.tistory.com/56)

Reply:
첨부해 주신 참고 블로그에 따르면 
passHref 를 넣지 않을 경우(`<Link>` 의 자식으로 커스텀 컴포넌트(`styled-component` 등)을 사용할때)
하위 컴포넌트로 href 속성이 전달되지 않는다는 거군요!

즉 passHref를 넣는 이유는 a 태그에 href가 빠져있는 것을 방지하기 위함이고,
UX, SEO 에 좋지 않은 영향을 주는 것을 막기 위함이네요!

Reply:
멋져용